docs/maintainers/concept-ci.md

```diff
@@ -0,0 +1,125 @@
+# Concept CI
```

Comment:
Perhaps we can make this a general "Continuous integration" document? That way, we can also include information about the link checker. What do you think?

Reply:
Good idea!